plantseg/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Initialise model registry at startup"""
 
+import torch  # required to patch #273
```

Comment:
Maybe you can try

```python
import torch    # noqa: F401; required to patch #273
```

or add this

```python
__all__ = [torch]
```

But run `pre-commit` to make sure. The dev env can be installed by `mamba env create -f environment-dev.yml`